ru/news/_posts/2015-06-25-rubyconftw-2015-call-for-proposals.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: news_post
+title: "Открыт прием заявок на RubyConf Taiwan 2015"
+author: "Juanito Fatas"
+translator: "gazay"
+date: 2015-07-14 15:30:00 +0000
+lang: ru
+---
+
+[RubyConf Taiwan 2015](http://rubyconf.tw) пройдет 11-12 сентября на
+тропическом острове в центре Азии – Тайпее, Тайвань.
+
+[Matz](https://twitter.com/yukihiro_matz),
+[Aaron Patterson](https://twitter.com/tenderlove),
+[Sarah Allen](https://twitter.com/ultrasaurus),
```

Comment:
I don't like transliteration of names. Also can we please discuss things in English as was asked by @stomar